app/views/simulator/edit.html.erb

```diff
@@ -32,6 +32,9 @@
       <li class="dropdown">
         <a href="#" class="" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project<span></span></a>
         <ul class="dropdown-menu">
+          <% if not @project.nil? %>
+             <li><a class="dropdown-item logixButton text-left pl-1" href="<%= user_project_url(@project.author, @project) %>"><%= @projectName %></a></li>
```

Comment:
Rather than @projectName, label it as `View Project Page`. Thanks!

Reply:
ok

Reply:
done